BackendNew/scripts/create-tables.sql

```diff
@@ -82,32 +80,37 @@ CREATE TABLE Student (
     DateOfStart DATE,
     Degree varchar(100),
     Password varchar(100),
```

Comment:
I have a doubt.  Suppose a student registered for a internship and could not complete it properly. Then he again register for internship. According to this "STUDENT" table he should have 2 accounts with same username(student ID) with different passwords.